run.sh

```diff
@@ -14,6 +14,9 @@ case "${1}" in
 		ENV="sbox"
     LONG_ENV="sandbox"
     CRITICALITY="Low"
+    LOG_NAME="hmcts-sbox-law"
```

Comment:
this should go to `hmcts-sandbox`

```suggestion
    LOG_NAME="hmcts-sandbox"
```